modules/ROOT/pages/header-inject-remove-task.adoc

```diff
@@ -1,54 +1,70 @@
-= Header Injection and Removal
+= Header Injection
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 [width="100%", cols="5,15"]
 |===
->s| Policy Name | Header injection/Header removal
->s|Summary      | Adds/removes headers to a request or a response
+>s| Policy Name | Header Injection
+>s|Summary      | Adds headers to a request or a response
 >s|Category | Transformation
 >s| First Mule version available | 4.1.0
 .1+>.^s| Returned Status Codes
 |
```

Comment:
This policy can return a 500 HTTP status code if the configuration includes an expression that fails to be evaluated.

Reply:
Done! Thanks!